app/graphql/mutations/create_street_art.rb

```diff
@@ -0,0 +1,22 @@
+class Mutations::CreateStreetArt < Mutations::BaseMutation
+  argument :latitude, String, required: false
+  argument :longitude, String, required: false
+  argument :address, String, required: false
+  argument :city, String, required: false
+  argument :state, String, required: false
+  argument :zipcode, String, required: false
+  argument :image_urls, String, required: true
+  argument :description, String, required: false
+  argument :artist_name, String, required: false
+  argument :art_name, String, required: false
+  argument :instagram_handle, String, required: false
+  argument :favorite, Boolean, required: false
+  argument :visited, Boolean, required: false
+  argument :user_id, Integer, required: true
+
+  type Types::StreetArtType
+
+  def resolve(latitude: nil, longitude: nil, address: nil, city: nil, state: nil, zipcode: nil, image_urls:, description: nil, artist_name: nil, art_name: nil, instagram_handle: nil, favorite: false, visited: false, user_id:)
```

Comment:
Setting default values to `nil` but some will not stay that way because they will be filled-in in the model